tests/callbacks/test_lambda_cb.py

```diff
@@ -15,15 +15,9 @@
 
 from pytorch_lightning import Trainer, seed_everything
 from pytorch_lightning.callbacks import LambdaCallback
-from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from tests.base.boring_model import BoringModel
 
 
-def test_lambda_raise_misconfiguration():
-    with pytest.raises(MisconfigurationException, match='does not exist in supported callbacks function'):
-        LambdaCallback(invalid=lambda *args: args)
-
-
```

Comment:
why did you remove this check?

Reply:
I added it again 👍

Reply:
Because `MisconfigurationException` no longer raise because `__init__` receives `Callable` objects as a real argument rather that `*kwargs`